### ISSUE TITLE
problem about versions

### ISSUE THREAD

**Issue by csufangyu** [upul/behavioral_cloning#5]
First thanks for your great work!I am very interesting for it.But when I run your example,I met many problems.and I found it may be the problem that versions of Python libraries.could you tell me the  version of Python libraries ?such as keras,tensorflow.
thank you very much!

**Comment by krishnaprasad1996** [comment]
yeah even im getting these issues..Dude pls tell us the version numbers
